graphblas/core/operator/binary.py

```diff
@@ -681,21 +713,22 @@ def _initialize(cls):
                     orig_op.gb_name,
                 )
                 new_op._add(cur_op)
-        # Add floordiv
-        # cdiv truncates towards 0, while floordiv truncates towards -inf
-        BinaryOp.register_new("floordiv", _floordiv, lazy=True)  # cast to integer
-        BinaryOp.register_new("rfloordiv", _rfloordiv, lazy=True)  # cast to integer
+        if _supports_udfs:
+            # Add floordiv
+            # cdiv truncates towards 0, while floordiv truncates towards -inf
+            BinaryOp.register_new("floordiv", _floordiv, lazy=True)  # cast to integer
+            BinaryOp.register_new("rfloordiv", _rfloordiv, lazy=True)  # cast to integer
```

Comment:
For operators that are available by default if UDFs are supported, should we give good error messages such as "numba required..."?

Reply:
Done.